### DISCUSSION TITLE
Analog line in possible?

### DISCUSSION THREAD

**Discussion by Maeckes89** [mikebrady/shairport-sync#1648] [Q&A]
Hi there,

is it possible, to get any line-in to stream it over airplay?

I would like to hear my vinyl-records all over the house and for this, I would like to use the line-out of my vinyl player (or vinyl pre-amp) and inject it into the airplay 2 network.

DIY is welcome but not necessary.

Cheers and thank you for any of your thoughts!

**Comment by mikebrady** [comment]
It's an attractive idea alright, but Shairport Sync is an AirPlay "sink", whereas what you are looking for an AirPlay "source" app. Perhaps the excellent [OwnTone](https://owntone.github.io/owntone-server) guys might have some ideas...
